packages/schematics/angular/server/files/application-builder/standalone-src/main.server.ts.template

```diff
@@ -1,7 +1,7 @@
 import { bootstrapApplication } from '@angular/platform-browser';
-import { App } from './app/app';
```

Comment:
Not sure if this is the only place we need to change. I see that there are 3 more `main.server.ts.template` files.

Reply:
It needs to be also replaced in;
- https://github.com/angular/angular-cli/blob/597af7f35d83b98fe8fc03e8f239ae2049b9651c/packages/schematics/angular/server/files/application-builder/ngmodule-src/app/app.module.server.ts.template#L3
- https://github.com/angular/angular-cli/blob/597af7f35d83b98fe8fc03e8f239ae2049b9651c/packages/schematics/angular/server/files/server-builder/ngmodule-src/app/app.module.server.ts.template#L5
- https://github.com/angular/angular-cli/blob/597af7f35d83b98fe8fc03e8f239ae2049b9651c/packages/schematics/angular/server/files/server-builder/standalone-src/main.server.ts.template#L2

Reply:
Done.